CHANGES.md

```diff
@@ -12,6 +12,7 @@ Change Log
   * `Camera.tilt` was deprecated in Cesium 1.6. It will be removed in Cesium 1.7. Use `Camera.pitch`.
   * `Camera.heading` and `Camera.tilt` were deprecated in Cesium 1.6. They will become read-only in Cesium 1.7. Use `Camera.setView`.
   * `Camera.setPositionCartographic` was deprecated in Cesium 1.6. It will be removed in Cesium 1.7. Use `Camera.setView`.
+   * The `eye`, `target`, and `up` parameters to `Camera.lookA`t were deprecated in Cesium 1.6. It will be removed in Cesium 1.8. Use the `target`, `offset`, and `transform` parameters.
```

Comment:
Submit an issue.  In general, just submit them when you open the pull request.  If we decide against them, we'll just close the issue.